### PR TITLE
13. GET /api

### PR DESCRIPTION
**Pull request by jake4369**
Complete endpoints.json, describing each endpoint and giving an example response

### CODE REVIEW

File: endpoints.json [hunk 1]
```diff
@@ -30,5 +30,78 @@
...
+      ]
+    }
+  },
+  "DELETE /api/comments/:commentId": {
+    "description": "deletes a specific comment",
+    "queries": [],
+    "exampleResponse": {
+      "status": 204,
+      "content": "no content"
+    }
```
Comment (Slyose) [inline endpoints.json:105]: You'll also need an endpoint to actually serve this data!

**Review by Slyose — CHANGES_REQUESTED** [review]
Data looks good, but you need an endpoint to actually send it!

File: __tests__/app.test.js [hunk 2]
```diff
@@ -633,6 +634,26 @@ describe("DELETE /api/comments/:comment_id", () => {
   });
 });
 
+describe("GET /api/endpoints", () => {
+  it("should respond with the correct JSON file", () => {
+    return request(app)
+      .get("/api/endpoints")
```
Comment (cjplatten) [inline __tests__/app.test.js:640]: The endpoint in the ticket is /api

**Review by cjplatten — COMMENTED** [review]
On comment to address

**Review by Geo-Beard — COMMENTED** [review]
Looks OK to me - feel free to merge and move on!

**Pull request merged**